fhir-model/src/main/java/com/ibm/fhir/model/util/FHIRUtil.java

```diff
@@ -797,36 +797,14 @@ public static boolean hasTag(Resource resource, Coding tag) {
         return updatedResource;
     }
 
-    // add for FHIRResource.java
-    private static final String BASIC_RESOURCE_TYPE_URL = "http://ibm.com/fhir/basic-resource-type";
-
     /**
-     * Returns the resource type (as a String) of the specified resource. For a virtual resource, this will be the
-     * actual virtual resource type (not Basic).
+     * Returns the resource type (as a String) of the specified resource. 
      * 
      * @param resource
      *            the resource
      * @return the name of the resource type associated with the resource
      */
     public static String getResourceTypeName(Resource resource) {
```

Comment:
should we deprecate this method?  and point to something in ModelSupport?

Reply:
sorry, missed this one before.  we discussed on our slack channel a bit and John also thought it should be deprecated, but I think his suggestion was just to use instance.getClass().getSimpleName() everywhere.  I sort of liked the idea of having the helper wrap this (even though its just one line), but I don't care much where it lives.

Reply:
No worries, I didn't think this was a big deal.  I like the helper method approach. FHIRUtil seems fine for now :)